[FIX] Clarify run indexing information for MRI acquisitions (#694)
bids-standard/bids-specification: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

If several scans of the same modality are acquired they MUST be indexed with the
[`run-<index>`](../99-appendices/09-entities.md#run) key-value pair:
`_run-1`, `_run-2`, `_run-3`, and so on (only nonnegative integers are allowed as
run labels).
If however these scans of the same modality contain different labels
(for example `_acq-`, `_ce-`, `_desc-`) then the run label and indexing are optional,
unless the labels are the same.

> When there is only one scan of a given type the run key MAY be

Reviewer: What about:

```Markdown
If several scans with the same acquisition parameters are acquired in the same session,
they MUST be indexed with the [`run-<index>`](../99-appendices/09-entities.md#run) entity:
`_run-1`, `_run-2`, `_run-3`, and so on (only nonnegative integers are allowed as
run labels).
If different entities apply,
such as a different session indicated by [`ses-<label>`](../99-appendices/09-entities.md#ses),
or different acquisition parameters indicated by [`acq-<label>`](../99-appendices/09-entities.md#acq),
then `run` is not needed to distinguish the scans and MAY be omitted.
```

This replaces the somewhat ambiguous concept of "modality" with acquisition parameters; in almost all cases identical acquisition parameters should imply the same modality.

This may also make the sentence "When there is only one scan of a given type the run key MAY be omitted." unnecessary, though I don't think that it harms clarity to keep it in.

Author: This wording sounds fine with me. I've gone ahead and made the changes, sorry for the delayed response.

If several scans of the same modality are acquired they MUST be indexed with the
[`run-<index>`](../99-appendices/09-entities.md#run) key-value pair:
`_run-1`, `_run-2`, `_run-3`, and so on (only nonnegative integers are allowed as

> run labels). If however these scans of the same modality contain different labels (e.g. `_acq-`, `_ce-`, `_desc-`) then the run label and indexing are optional, unless the labels are the same. When there is only one scan of a given type the run key MAY be

Reviewer 2: Thanks for the PR @dlevitas I suggest to break lines at reasonable places to improve future git diff, and to replace "e.g." by more readable "for example", because we are trying to reduce the use of latin in the BIDS specification where possible.

```suggestion
run labels).
If however these scans of the same modality contain different labels
(for example `_acq-`, `_ce-`, `_desc-`) then the run label and indexing are optional,
unless the labels are the same.
When there is only one scan of a given type the run key MAY be
```

> If several scans of the same modality are acquired they MUST be indexed with the
> [`run-<index>`](../99-appendices/09-entities.md#run) key-value pair:
> `_run-1`, `_run-2`, `_run-3`, and so on (only nonnegative integers are allowed as

Reviewer: ```suggestion
```

> run labels).

Reviewer: ```suggestion
```

If several scans with the same acquisition parameters are acquired in the same session,
they MUST be indexed with the [`run-<index>`](../99-appendices/09-entities.md#run) entity:
`_run-1`, `_run-2`, `_run-3`, and so on (only nonnegative integers are allowed as
run labels).

If different entities apply,
such as a different session indicated by [`ses-<label>`](../99-appendices/09-entities.md#ses),
or different acquisition parameters indicated by [`acq-<label>`](../99-appendices/09-entities.md#acq),

> then `run` is not needed to distinguish the scans and MAY be omitted.

Reviewer 3: I think this is crucial information that should be reflected in the run entity definition, rather than in a specific modality's section.
